src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -27,6 +36,7 @@ public ActionFactory(final UserLookupService userLookupService,
         this.userLookupService = userLookupService;
         this.customFieldParser = customFieldApiParser;
         this.config = config;
+        this.dltStatuses = config.getDeliveryLeadTimeFields().split(",");
```

Comment:
Thanks for moving this to a config. But it might be cleaner to build this data structure in the main() method and just store the array (or the `Set<>`) in the config instead.